src/hooks/useShifts.tsx

```
@@ -259,4 +261,327 @@ export const useDeleteShift = () => {
+    },
+    onMutate: async ({ shift, scope, restaurantId }) => {
+      await queryClient.cancelQueries({ queryKey: ['shifts', restaurantId] });
+
+      const previousData = queryClient.getQueriesData<Shift[]>({ queryKey: ['shifts', restaurantId] });
+      const parentId = getSeriesParentId(shift);
+      const shiftStartTime = new Date(shift.start_time).getTime();
+
+      queryClient.setQueriesData<Shift[]>({ queryKey: ['shifts', restaurantId] }, (old) => {
+        if (!old) return old;
+
+        return old.filter((s) => {
```

Biome (2.3.13)

[error] This callback passed to filter() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)